src/main/java/org/cascadebot/cascadebot/commands/music/PlayingCommand.java

```diff
@@ -0,0 +1,64 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import lavalink.client.player.IPlayer;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.FormatUtils;
+
+import java.util.Set;
+
+public class PlayingCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        IPlayer player = context.getData().getMusicPlayer().getPlayer();
+        AudioTrack track = player.getPlayingTrack();
+
+        if (player.getPlayingTrack() == null) {
+            context.getTypedMessaging().replyWarning("No music playing!");
+        } else {
+            EmbedBuilder embedBuilder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+            embedBuilder.setTitle(track.getInfo().title + " - " + track.getInfo().author, track.getInfo().uri);
+            embedBuilder.setThumbnail("https://img.youtube.com/vi/" + track.getIdentifier() + "/hqdefault.jpg");
```

Comment:
you don't always have youtube tracks........

Reply:
Oof true

Reply:
I suggest we make a util method somewhere that uses the uri to get what service it is from and therefore what image url to use. Perhaps `MusicUtils`?

Reply:
I'll look into what services we can get an image from, it might not be worth it if youtube is the only one.

Reply:
Well if YouTube is the only one then we can just do it for that

Reply:
Feels like a waste of a util method.

Reply:
Well we could probably just make it into logic in the command if it's just youtube although I doubt it will be just YouTube